media/admin/com_virtuemart/models/zasilkovna_src/VirtueMartModelZasilkovna/Carrier/VendorGroups.php

```diff
@@ -5,12 +5,11 @@
 class VendorGroups
 {
     // constant values are used in zasilkovna.xml
-    const ALZABOX = 'alzabox';
-    const ZPOINT = 'zpoint';
+	const ZPOINT = 'zpoint';
     const ZBOX = 'zbox';
 
     static private $countryCodeMapping = [
```

Comment:
všechny prvky mají nyní stejné hodnoty,  není to mapování země -> typ výdejního místa již zbytečné ?

Reply:
Podle me ne, ta fce zaroven resi country kody, kde zadni vendori nejsou

Reply:
a proč kvůli country kódu musíme mít mapování, kdy každý country má kód má pole , které má stejné parametry pro všechny země ?